cells/std/nixago/editorconfig.nix

```diff
@@ -2,10 +2,19 @@
   inputs,
   cell,
 }: let
+  l = nixpkgs.lib // builtins;
   inherit (inputs) nixpkgs;
 in {
   configData = {};
   output = ".editorconfig";
-  format = "ini";
+  engine = request: let
+    inherit (request) configData output;
+    name = l.baseNameOf output;
+    value = {
+      globalSection = {root = configData.root or false;};
+      sections = l.removeAttrs configData ["root"];
```

Comment:
Interesting -- this reminds me of an oddity I've noticed in a `treefmt.toml` generated with Standard:

```toml
[formatters] # <- unnecessary, but harmless (i think)
[formatters.nix]
command = "alejandra"
...
```

That said, the additional section could have been something I introduced in my own project accidentally...

So I take it this removal of the `root` section prevents something similar?

Reply:
Yeah, this is just the way that nix' `toTOML` is implemented.